src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/PublicationViewModels.cs

```diff
@@ -11,42 +11,49 @@ public class PublicationViewModel
     {
         public Guid Id { get; set; }
 
-        public string Title { get; set; }
+        public string Title { get; set; } = Empty;
 
-        public string Summary { get; set; }
+        public string Summary { get; set; } = Empty;
 
-        public string Slug { get; set; }
+        public string Slug { get; set; } = Empty;
```

Comment:
Not sure. Even if not, I think these endpoints are meant to be used generally rather than for one specific purpose. I'll defer to Nick on this.

Reply:
Don't think they're being used in the dashboard, but slugs are useful as they can be used for links to the public frontend and consequently should definitely be in the API.

Just a reminder that we should be designing REST APIs to be used more generally rather than for specific use-cases/pages. Part of why we're having to do this all this refactoring is because we went down the latter approach of tailoring requests for the current admin dashboard.